components/datetime/src/provider/calendar/skeletons.rs

```diff
@@ -18,10 +18,10 @@ use litemap::LiteMap;
     feature = "provider_serde",
     derive(serde::Serialize, serde::Deserialize)
 )]
-#[zerofrom(cloning_zf)]
 pub struct DateSkeletonPatternsV1<'data>(
     #[cfg_attr(feature = "provider_serde", serde(borrow))]
-    pub  LiteMap<SkeletonV1, PatternPlurals<'data>>,
+    #[zerofrom(clone)]
+    pub LiteMap<SkeletonV1, PatternPlurals<'data>>,
```

Comment:
q: any reason why ZeroMapping this is hard?

Reply:
It's probably not now that we have the derive.